interview-questions.md

```diff
@@ -9,59 +9,82 @@ Then, research the question to expand on your answer. Even if you feel you have
+
+  - Reference: [React Introduction](https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/react/intro.md)
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: From my understanding, the render method is required in a React class component. The render method is an important part in displaying the contents of a web application. The render method also rerenders the page after every state change within the componenet.
 
-  Researched answer:
+  Researched answer: render() is similar to the built-in JS function return. render() returns the content inside of the parentheses which is JSX. The render method contains the return() method and JSX lives inside the return() method.
 
+  - Reference: [React Introduction](https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/react/intro.md)
 
 
 4. What is JSX? What is one syntax difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX stands for JavaScript XML. JSX consists of JavaScript and something very similar to HTML and its use case is to add functionality using JavaScript directly into element tags that are based on HTML tags. One difference between HTML and JSX are attribute names. For example, when I worked with JSX, I had to get used to camel case since JSX uses this type of naming convention on their attributes. For example, a commonly used attribute in HTML is `class`. This same feature in JSX is `className`.
 
-  Researched answer:
+  Researched answer: JSX is JavaScript flavored HTML that behaves almost like HTML. The biggest difference is that JSX uses JavaScript directly in what looks like HTML to add logic and display data
 
+  - Reference: [React Introduction](https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/react/intro.md)
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: yarn is a software that acts as a package manager. The purpose of a package manage is to be able to install packages which contain useful code snippets and JavaScript code that has been made from other developers. Other developers are able to utilize these packages which can greatly expedite development. One of the main use cases I've used yarn for is to create React applications. With just one terminal command, yarn install all of the dependencies needed to create a React application. Running this command creates some files such as: node_modules, package.json, and yarn.lock
 
+  Researched answer: Yarn is a package manager for your code that allows you to use and share JavaScript code with other developers. Yarn offers developers' solutions to different problems, making it easier for users to develop their software.
```

Comment:
Remember these answers need to be in your words. I appreciate that you cite your resources but you won't be doing that in an interview. 

The goal is that you are internalizing what you have researched not showing me your google prowess. Last time I accept something this close to the source material.

Reply:
Sorry I guess I misunderstood what the researched answer should entail. I thought I was supposed to have the more concise answer. But I think I remember you saying now that the researched answer should be a revision of 'your answer' portion